community/mutt/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --sysconfdir=/etc \
+    --with-screen=ncurses
```

Comment:
It's worth testing without this line too. :)

Reply:
The "ncurses line" ?

Reply:
Yeah.

Reply:
Done! I realize, you are a real specialist :-)